src/pkg/build/conveyorPacker_debootstrap_test.go

```diff
@@ -33,8 +38,12 @@ func TestDebootstrapConveyor(t *testing.T) {
 
 	err := dc.Get(testDef)
 	if err != nil {
+		//clean up tmpfs since assembler isnt called
+		os.RemoveAll(dc.tmpfs)
 		t.Fatalf("Debootstrap Get failed: %v", err)
 	}
+
+	os.RemoveAll(dc.tmpfs)
```

Comment:
Rather than these two `os.RemoveAll()`, you could `defer os.RemoveAll()` right after the `dc.Get()` line (before the conditional.)